rss-client/src/app/Cart/shopping-cart/shopping-cart.component.html

```diff
@@ -103,11 +98,12 @@ <h4>x{{cItem.quantity}}</h4>
                                 </td>
                                 <td class="align-bottom"><b>{{product.name}}</b></td>
                                 <td class="align-bottom" [ngStyle]="{'text-decoration':product.discounted == true?'line-through':'null'}">{{product.unitPrice}} Points</td>
-                                <td class="align-bottom" *ngIf="product.discounted">{{product.unitPrice - product.discountedAmount}} Points</td>
+                                <td class="align-bottom" *ngIf="product.discounted">{{product.unitPrice - (product.unitPrice * (product.discountedAmount*0.01))}} Points</td>
 
                             </tr>
                         </table>
-                    </div>
+                    </div> -->
+
```

Comment:
is there a reason to leave this in if it is commented out?

Reply:
That was from inherited code; it has been removed.